react-native-reanimated.d.ts

```diff
@@ -501,10 +501,19 @@ declare module 'react-native-reanimated' {
     interval: number;
   };
 
+  export type AnimatedSensor = {
+    sensor: SensorValue3D | SensorValueRotation | null;
+    unregister: () => void;
+    isAvailable: boolean;
+    config: {
```

Comment:
Could be using the type defined above
```suggestion
    config: SensorConfig
```